packages/display/src/Container.ts

```diff
@@ -479,6 +480,26 @@ export class Container extends DisplayObject
         this._bounds.updateID = this._boundsID;
     }
 
+    public getLocalBounds(rect?: Rectangle, noTransformSideEffects = true): Rectangle
```

Comment:
You override public method.
JSDoc is required.

`noTransformSideEffects` what this is mean and why this is `true` by default?

I think it is must be `skipUpdateTransfrom` how in getBounds method.

Reply:
As I indicated before, this is just a sketch PR. That is why no JSDoc yet.

Reply:
I removed that parameter.